sql/core/src/test/scala/org/apache/spark/sql/execution/SortSuite.scala

```diff
@@ -18,9 +18,13 @@
 package org.apache.spark.sql.execution
 
 import org.apache.spark.sql.catalyst.expressions.{BoundReference, Ascending, SortOrder}
+import org.apache.spark.sql.catalyst.dsl.expressions._
+
+import org.apache.spark.sql.test.TestSQLContext
 import org.apache.spark.sql.types.{IntegerType, StringType}
 
 class SortSuite extends SparkPlanTest {
+  import TestSQLContext.implicits.localSeqToDataFrameHolder
```

Comment:
On second thought, perhaps SparkPlanTest should have an implicit alias to this method so each subclass does not have to worry about importing it.